src/patternfly/components/Table/table.scss

```diff
@@ -143,6 +143,10 @@
   width: 100%;
   background-color: var(--pf-c-table--BackgroundColor);
 
+  td {
+    word-break: break-all;
```

Comment:
Actually Lauren will you change this to "word-break: break-word"

I tested this in the table, and some words were breaking when they shouldn't. Using "break-all" breaks every word, but "break-word" breaks just the really long words.

This is with **break-all** : 
<img width="376" alt="Screen Shot 2019-06-14 at 10 44 50 AM" src="https://user-images.githubusercontent.com/20118816/59517453-9e5fee80-8e91-11e9-96a2-2ffdda2c832c.png">


This is with "break-word":
<img width="363" alt="Screen Shot 2019-06-14 at 10 44 43 AM" src="https://user-images.githubusercontent.com/20118816/59517435-999b3a80-8e91-11e9-949f-e06776c59700.png">

Reply:
Changed it!